pkg/cmd/repo/create/http.go

```diff
@@ -236,6 +238,76 @@ func resolveOrganizationTeam(client *api.Client, hostname, orgName, teamSlug str
 	return &response, err
 }
 
+func listRepositoryTemplates(client *http.Client, hostname, owner string) (*list.RepositoryList, error) {
+	ownerConnection := "repositoryOwner(login: $owner)"
+
+	variables := map[string]interface{}{
+		"perPage": githubv4.Int(100),
+		"owner":   githubv4.String(owner),
+	}
+	inputs := []string{"$perPage:Int!", "$endCursor:String", "$owner:String!"}
+
+	type result struct {
+		RepositoryOwner struct {
+			Login        string
+			Repositories struct {
+				Nodes      []api.Repository
+				TotalCount int
+				PageInfo   struct {
+					HasNextPage bool
+					EndCursor   string
+				}
+			}
+		}
+	}
+
+	query := fmt.Sprintf(`query RepositoryList(%s) {
+		%s {
+			login
+			repositories(first: $perPage, after: $endCursor, ownerAffiliations: OWNER, orderBy: { field: PUSHED_AT, direction: DESC }) {
```

Comment:
Since the `gh repo list` command sorts the repository list by PUSHED_AT, I thought it would be more consistent to sort the template repository list by PUSHED_AT as well. 

https://github.com/cli/cli/blob/3443a752a9949ee4f954a84f7ab94a4b99df1b27/pkg/cmd/repo/list/http.go#L80-L89

Reply:
Seems reasonable to me. Since this is interactive, we don't have any backwards compatibility concerns here about the order anyway.